### ISSUE TITLE
Monitor scheduled jobs?

### ISSUE THREAD

**Issue by sburns** [ui/rq-scheduler#109]
Does anyone have any advice about how to monitor scheduled jobs?

**Comment by selwin** [comment]
You can use an RQ monitoring tool like Django RQ: https://github.com/ui/django-rq

**Comment by sburns** [comment]
I'm probably being dense but I don't see in what view django_rq will expose scheduled jobs (which are kept in a different redis key than started, finished or deferred jobs). I may work on a PR to django_rq if it would be helpful.

**Comment by selwin** [comment]
You can monitor the jobs from Django's admin interface.

**Comment by smaccona** [comment]
@selwin - Thanks for your work on `rq-scheduler` - extremely useful!

I'm in the same boat as @sburns with respect to monitoring - I have the following installed:

```bash
$ pip freeze | grep rq
django-rq==0.9.0
django-suit-rq==1.0.1
rq==0.5.6
rq-scheduler==0.6.0
```

In the Django Admin interface, I can see columns for Queued, Active, Deferred and Finished jobs, but nothing for scheduled (apologies, I had to blank out the queue names):

![image](https://cloud.githubusercontent.com/assets/394952/14833398/1b98cb1a-0bcd-11e6-84c0-8507d451a19a.png)

I created a couple of test jobs using `enqueue_at` and I can see them in the Django shell:

```python
>>> import django_rq
>>> s = django_rq.get_scheduler()
>>> s.get_jobs()
[Job(u'5fe50431-42ce-4424-a2d1-6993b1f9d9e0', enqueued_at=None), Job(u'24a83f48-ad2c-4477-9c8b-730a119714b7', enqueued_at=None)]
>>>
```

But all the counts in the admin are still 0. Where in the Django admin can we see jobs that are scheduled? Thanks!
